src/example_onchain_graph.py

```diff
@@ -0,0 +1,13 @@
+import asyncio
```

Comment:
This example file should not be placed with the source files. It deserves another folder for itself.

e.g.

`examples/onchain_graph.py`

Reply:
Moved all examples under `examples` folder as suggested